test/helpers/utils.go

```diff
@@ -529,3 +529,15 @@ func SplitContainerURL(url string) (registry, image, tag string, success bool) {
 
 	return registry, image, tag, true
 }
+
+// GetLatestImageVersion infers which docker tag should be used
+func GetLatestImageVersion() string {
```

Comment:
Could this wrap `SplitContainerURL` just above?

Reply:
Lol, totally missed it